content/maxwell1_fundamentals/index.rst

```diff
@@ -5,34 +5,36 @@ Maxwell I: Fundamentals
+
+    - :ref:`Interface Conditions<maxwell1_fundamentals_interface_conditions_index>`: To solve Maxwell's equations in general cases, we must consider the conditions on fields and fluxes at physical interfaces. Here, the appropriate interface conditions for :math:`\mathbf{e}`, :math:`\mathbf{h}`, :math:`\mathbf{j}`, :math:`\mathbf{d}` and :math:`\mathbf{b}` are presented.
+
+    - :ref:`Electromagnetic Wave Equation (Time Domain)<time_domain_equations>`: Here, we show that within a homogeneous media, time-dependent electromagnetic signals behave according to the diffusive wave equation. Diffusive and wave behaviours are very important when considering time-domain electromagnetic methods.
+
+    - :ref:`Electromagnetic Wave Equation (Frequency Domain)<frequency_domain_equations>`: Here, we show that within a homogeneous media, frequency-dependent electromagnetic signals behave according to the vector Helmholtz equation. The physical understanding of this equation is very important when considering frequency-domain electromagnetic methods.
+
+    - :ref:`Plane Wave in Homogeneous Media<plane_waves_in_homogeneous_media_index>`: Here, we expand on the content provided in the previous two sections. For plane waves in the time domain, we discuss diffusion distance, peak time and peak velocity. In the frequency domain, we discuss the skin depth, phase velocity, wavelength and apparent resistivity. Numerical apps are also provided to visualize plane waves in various cases.
+
+    - :ref:`Reflection and Refraction of Plane Waves<reflection_and_refraction_index>`: When plane waves reach an interface characterized by an abrupt change in physical properties, the signal is altered. Here, the physics pertaining to plane waves as they reach an interface is presented. Content includes: Snell's law, the Fresnel equations and Brewster's angle. Numerical apps are provided to simulate electromagnetic signals.
+
+    - :ref:`Maxwell's Equations with Electromagnetic Sources<maxwell_fundamentals_sources>`: For applications in electromagnetic geoscience, a source is frequently used to generate electromagnetic signals. Here, we demonstrate how Maxwell's equations must be altered to accommodate electromagnetic sources.
+
+    - :ref:`Dipole Sources in Homogeneous Media<dipole_sources_in_homogeneous_media_index>`: The fields generated by dipole sources and of great fundamental important to electromagnetic geoscience, as they do very well at approximating the electromagnetic sources used for many geophysical applications. Here, the fields generated by dipole sources within a homogeneous medium are presented. Numerical apps are provided for visualizing the fields.
+
+    - :ref:`Solving Maxwell's Equations in Practice<solving_maxwells_equations>`: Here, we discuss what is needed to solve Maxwell's equations in practice.
 
 **Contents**
```

Comment:
regarding titles: to tighten things up, here is my suggestion, see what you think (cc @dougoldenburg ) 

![image](https://cloud.githubusercontent.com/assets/6361812/24690053/68ddbe88-197f-11e7-9d9f-6a1914857dcb.png)